ferry.go

```diff
@@ -917,8 +917,9 @@ func (f *Ferry) Progress() *Progress {
 	for _, completedPaginationKey := range serializedState.LastSuccessfulPaginationKeys {
 		completedPaginationKeys += completedPaginationKey
 	}
-
-	s.ETA = (time.Duration(math.Ceil(float64(totalPaginationKeysToCopy-completedPaginationKeys)/estimatedPaginationKeysPerSecond)) * time.Second).Seconds()
+	var remainingPaginationKeys float64 = 0
+	remainingPaginationKeys = math.Max(0, float64(totalPaginationKeysToCopy-completedPaginationKeys))
```

Comment:
Can you make a comment inline on why this is possible? iirc it's possible because we can overshoot with the batching, if rows are inserted between when ghostferry is started and when all the copy is completed.